api/app/models/model.py

```diff
@@ -18,26 +19,29 @@ def get_all(table):
     column = get_columns(table)
     results = list()
     try:
-        db.execute("SELECT * FROM "+table)
+        query = "SELECT * FROM {}".format(table)
+        db.prepare(query)
+        db.execute(query)
```

Comment:
ini kenama mesti inser querynya lagi yh kalau udh prepare??

```
cur = conn.cursor(cursor_factory=PreparingCursor)
cur.prepare(
    "select * from tables where i = %s and j = %s")
for i, j in parameters:
    cur.execute((i, j))
    for record in cur:
        do_something_with(record)
```

Reply:
```
db.prepare("SELECT * FROM %s")
db.execute((table))
```

Reply:
solve in https://github.com/BiznetGIO/RESTKnot/pull/62/commits/c4fe2ecca0476ff442002f6cc6e82f8297237f21